code/modules/mob/living/carbon/human/species.dm

```diff
@@ -120,11 +189,26 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	if(!GLOB.roundstart_races.len)
 		GLOB.roundstart_races += "human"
 
+/**
+  * Checks if a species is eligible to be picked at roundstart.
+  *
+  * Checks the config to see if this species is allowed to be picked in the character setup menu.
+  * Used by [proc/generate_selectable_species].
+  */
 /datum/species/proc/check_roundstart_eligible()
 	if(id in (CONFIG_GET(keyed_list/roundstart_races)))
 		return TRUE
 	return FALSE
 
+/**
+  * Generates a random name for a carbon.
+  *
+  * This generates a random unique name based on a human's species and gender.
+  * Arguments:
+  * gender - The gender that the name should adhere to. Use MALE for male names, use anything else for female names.
+  * unique -
```

Comment:
Note to self: Find out what unique does and add it to this documentation

Reply:
It tries to ensure that the name is unique and no dupes exist in the station rn